### PR TITLE
🌙 Add GCP service accounts and keys MQL resources

### PR DESCRIPTION
**Pull request by imilchev**
- `gcp.project.iam.serviceAccounts{*}`
- `gcp.project.iam.serviceAccounts{keys{*}}`

### CODE REVIEW

File: resources/packs/gcp/gcp.lr [hunk 1]
```diff
@@ -1567,3 +1569,51 @@ private gcp.project.loggingservice.bucket.indexConfig {
...
+  serviceAccounts() []gcp.project.iamService.serviceAccount
+}
+
+// GCP Service Account
+private gcp.project.iamService.serviceAccount {
+  // Project ID
+  projectId string
+  // Service account name
+  name string
+  // Unique, stable numeric ID for the service account
```
Comment (misterpantz) [inline resources/packs/gcp/gcp.lr]:
```suggestion
  // Unique, stable, numeric ID for the service account
```

File: resources/packs/gcp/gcp.lr [hunk 2]
```diff
@@ -1567,3 +1569,51 @@ private gcp.project.loggingservice.bucket.indexConfig {
...
+  description string
+  // OAuth 2.0 client ID
+  oauth2ClientId string
+  // Whether the service account is disabled
+  disabled bool
+  // Service account keys
+  keys() []gcp.project.iamService.serviceAccount.key
+}
+
+// GCP Service account keys
```
Comment (misterpantz) [inline resources/packs/gcp/gcp.lr]:
```suggestion
// GCP service account keys
```

File: resources/packs/gcp/gcp.lr [hunk 3]
```diff
@@ -1567,3 +1569,51 @@ private gcp.project.loggingservice.bucket.indexConfig {
...
+private gcp.project.iamService.serviceAccount.key {
+  // Service account key name
+  name string
+  // Algorithm (an possibly key size) for the key
```
Comment (misterpantz) [inline resources/packs/gcp/gcp.lr]: Do you mean **and** possibly key size? If so, I recomment:

```suggestion
  // Algorithm (and possibly size) of the key
```

**Review by misterpantz — COMMENTED** [review]
Beautiful LR! Thanks so much for such careful attention to the style we're going for. I have just a couple suggestions.

**Review by chris-rock — APPROVED** [review]
Great addition @imilchev

**Pull request merged**